app/main.py

```diff
@@ -15,5 +15,14 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
-
-# write your code here
+sorted_variables = {"mutable": [], "immutable": []}
+mutable = set, dict, list
+immutable = int, float, str, tuple, bool
+variables = [lucky_number, pi, one_is_a_prime_number, name,
+             my_favourite_films, profile_info, marks, collection_of_coins]
+for new_var in variables:
+    if type(new_var) in mutable:
+        sorted_variables["mutable"].append(new_var)
+    else:
+        sorted_variables["immutable"].append(new_var)
+print(sorted_variables)
```

Comment:
You did the task correctly, but overcomplicated it) All that is needed is to declare `sorted_variables` dictionary and directly pass here mutable and immutable variables. No need to do extra logic)